lib/gpfpiso.gi

```diff
@@ -2812,7 +2820,9 @@ local d,f,group,act,g,sy,b,c,borel,weyl,a,i,iso,ucs,gens,gl;
       iso:=SplitBNRewritingPresentation(group,borel,weyl,true);
       return IsomorphismGroups(G,Source(iso))*iso;
     else
-      Error("can't do yet");
+      borel:=Group(SpecialPcgs(borel));
+      iso:=SplitBNRewritingPresentation(group,borel,weyl,true);
+      return IsomorphismGroups(G,Source(iso))*iso;
```

Comment:
This code is not reached by CI, is it possible to add a test case for it with reasonable effort (i.e. not too big and slow)?

Reply:
The generic test would be basically to run through simple groups up to order 10 million and to call `IsomorphismFpGroupForRewriting`, but that will take very long.

What the code actually tries to do is to compute a BN-pair for the group through properties, rather than by starting to construct the group by definition as Lie type group. This works well for PSL, and gets increasingly worse the more the group differs from Chevalley type A.
I would hope that ultimately it all could be replaced with proper Chevalley code.